src/QMCTools/qmcfinitesize.cpp

```diff
@@ -57,39 +57,44 @@ int main(int argc, char** argv)
   std::cout.precision(12);
 
   std::unique_ptr<SkParserBase> skparser(nullptr);
-  int iargc = 2;
 
-  while (iargc + 1 < argc)
+  if (argc<4)
```

Comment:
Add a comment before this like to help understanding the code. Also clang-format the change if you can.

Reply:
Addressed.